server/amp/handlers/story-page.js

```diff
@@ -67,8 +67,10 @@ async function ampStoryPageHandler(
     )
       return res.redirect(story.url);
 
+    const timezone = get(additionalConfig, ["publisher", "timezone"], null);
```

Comment:
default timeZone is null?

Reply:
Yes, the default date will look something like this `2021-07-12T03:51:41Z`